public/css/paper.css

```
@@ -0,0 +1,473 @@
+  font-weight: 700;
+  text-decoration: underline;
+  text-decoration-color: var(--accent-border);
+  text-decoration-thickness: 0.06em;
+  text-underline-offset: 0.2em;
+  background: 0;
+  border: 0;
+  border-radius: 0;
+  box-shadow: none;
+  padding: 0;
+  cursor: pointer;
+  font: inherit;
```

Stylelint (17.10.0)

[error] Overridden property "font-weight" by shorthand "font" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)